src/main/java/com/internet/shop/db/Storage.java

```diff
@@ -1,16 +1,43 @@
 package com.internet.shop.db;
 
+import com.internet.shop.model.Order;
 import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.model.User;
 import java.util.ArrayList;
 import java.util.List;
 
 public class Storage {
-    public static List<Product> products = new ArrayList<Product>();
+    public static List<Product> products = new ArrayList<>();
+    public static List<User> users = new ArrayList<>();
+    public static List<ShoppingCart> shoppingCarts = new ArrayList<>();
+    public static List<Order> orders = new ArrayList<>();
     private static Long productId = 0L;
+    private static Long userId = 0L;
+    private static Long shoppingCartId = 0L;
+    private static Long orderId = 0L;
 
     public static void addProduct(Product product) {
         productId++;
         product.setId(productId);
         products.add(product);
     }
+
+    public static void addUser(User user) {
+        userId++;
+        user.setId(userId);
```

Comment:
```suggestion
        user.setId(++userId);
```
same in other methods

Reply:
done